sdk/servicebus/azure-messaging-servicebus-stress/README.md

```diff
@@ -0,0 +1,74 @@
+# Azure Service Bus Stress test client library for Java
+
+Represents stress tests for Service Bus client.
+
+## Getting started
+
+The stress tests for service bus client is developed from [azure-sdk-chaos][azure_sdk_chaos]. 
+
+Follow the same startup steps in [Event Hubs Stress Test - Getting started][event_hubs_stress_test_start].
+
+## Key concepts
+
+### Project Structure
+
+See [Layout][stress_test_layout] section for details. 
+
+Below is the current structure of project:
+```
+.
+├── src/                         # Test code
+├── templates/                   # A directory of helm templates that will generate Kubernetes manifest files.
+├── Chart.yaml                   # A YAML file containing information about the helm chart and its dependencies
+├── Dockerfile                   # A Dockerfile for building the stress test image
+├── stress-test-resouce.bicep    # An Azure Bicep for deploying stress test azure resources
+├── values.yaml                  # Any default helm template values for this chart, e.g. a `scenarios` list
+├── pom.xml
+└── README.md
+```
+
+### Cluster Namespace 
+
+Similar with concepts in [Event Hubs Stress Test - Cluster Namespace][event_hubs_stress_test_namespace].
+
+For Service Bus, the default value we set is `java-sb`.
+
+## Examples
+
+### Test Queue/Topic
+
+You can switch to test Queue or Topic by providing the program argument `--SERVICEBUS_ENTITY_TYPE=QUEUE/TOPIC`. 
+
+If you haven't provided, the default value we are using is `QUEUE`.
+
+### Add New Test Scenario
+
+See steps in [Event Hub Stress Test - Add New Test Scenario][event_hubs_stress_test_add_test].
+
+The difference is that here we extend `ServiceBusScenarios` for new test scenarios.
+
+### Add New Scenario Option
+
+See steps in [Event Hub Stress Test - Add New Scenario Option][event_hubs_stress_test_add_option].
```

Comment:
Service Bus?

Reply:
Same with one above.

Reply:
Now I copied same content to SB rather than use the EH links.